### PR TITLE
chore: release main

### PR DESCRIPTION
**Pull request by origamiserviceuser**
:robot: I have created a release \*beep\* \*boop\*
---
<details><summary>@financial-times/o-cookie-message: 6.6.0</summary>

### Features

* o-cookie-message, publish tsx template ([#1199](https://www.github.com/Financial-Times/origami/issues/1199)) ([7052fe3](https://www.github.com/Financial-Times/origami/commit/7052fe34bef7f96ed867dc79409522fb29a22017))
</details>
<details><summary>@financial-times/o-forms: 9.11.2</summary>

### Bug Fixes

* trigger release to rollback o-forms docs ([258d573](https://www.github.com/Financial-Times/origami/commit/258d5736452f9802691733175e0d1254cab36409))
</details>
<details><summary>@financial-times/o-labels: 6.5.6</summary>

### Bug Fixes

* correct table syntax ([86ff070](https://www.github.com/Financial-Times/origami/commit/86ff070848b001751bf36b67b83b5d1c4567a6f8))
</details>
<details><summary>@financial-times/o-social-follow: 1.0.7</summary>

### Bug Fixes

* ensure o-social-follow docs remain published to registry ([ccd9d3c](https://www.github.com/Financial-Times/origami/commit/ccd9d3cf4192aa582071b12dfc61de43cc06c2d3))
* preserve o-social-follow docs ([e6a3341](https://www.github.com/Financial-Times/origami/commit/e6a3341a6abb982683e25c092d3b7aaac9a48eda))
</details>

This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

### CODE REVIEW

File: components/o-social-follow/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+### [1.0.7](https://www.github.com/Financial-Times/origami/compare/o-social-follow-v1.0.6...o-social-follow-v1.0.7) (2023-07-20)
+
+
+### Bug Fixes
+
+* ensure o-social-follow docs remain published to registry ([ccd9d3c](https://www.github.com/Financial-Times/origami/commit/ccd9d3cf4192aa582071b12dfc61de43cc06c2d3))
+* preserve o-social-follow docs ([e6a3341](https://www.github.com/Financial-Times/origami/commit/e6a3341a6abb982683e25c092d3b7aaac9a48eda))
```
Comment (notlee) [inline components/o-social-follow/CHANGELOG.md:9]:
```suggestion
* ensure o-social-follow design guidelines remain published to registry ([ccd9d3c](https://www.github.com/Financial-Times/origami/commit/ccd9d3cf4192aa582071b12dfc61de43cc06c2d3))
```

**Review by notlee — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
:robot: Release for @financial-times/o-cookie-message is at https://github.com/Financial-Times/origami/releases/tag/o-cookie-message-v6.6.0 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release for @financial-times/o-forms is at https://github.com/Financial-Times/origami/releases/tag/o-forms-v9.11.2 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release for @financial-times/o-labels is at https://github.com/Financial-Times/origami/releases/tag/o-labels-v6.5.6 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release for @financial-times/o-social-follow is at https://github.com/Financial-Times/origami/releases/tag/o-social-follow-v1.0.7 :sunflower:

**Pull request merged**